test-suite/src/test/scala/org/scalajs/testsuite/javalib/ConcurrentSkipListSetTest.scala

```diff
@@ -0,0 +1,419 @@
+/*                     __                                               *\
+**     ________ ___   / /  ___      __ ____  Scala.js Test Suite        **
+**    / __/ __// _ | / /  / _ | __ / // __/  (c) 2013-2015, LAMP/EPFL   **
+**  __\ \/ /__/ __ |/ /__/ __ |/_// /_\ \    http://scala-js.org/       **
+** /____/\___/_/ |_/____/_/ | |__/ /____/                               **
+**                          |/____/                                     **
+\*                                                                      */
+package org.scalajs.testsuite.javalib
+
+import scala.language.implicitConversions
+
+import scala.collection.JavaConversions._
+import scala.collection.mutable
+
+import org.scalajs.jasminetest.JasmineTest
+
+import scala.scalajs.runtime.UndefinedBehaviorError
+
+import java.util.concurrent.ConcurrentSkipListSet
+import java.util.Comparator
+
+object ConcurrentSkipListSetTest extends JasmineTest {
+
+  describe("java.util.concurrent.ConcurrentSkipListSet") {
+
+    it("should store and remove ordered integers") {
+      val csls = new ConcurrentSkipListSet[Int]()
+
+      expect(csls.size()).toEqual(0)
+      expect(csls.add(222)).toBeTruthy
+      expect(csls.size()).toEqual(1)
+      expect(csls.add(111)).toBeTruthy
+      expect(csls.size()).toEqual(2)
+      expect(csls.first).toEqual(111)
+      expect(csls.remove(111)).toBeTruthy
+
+      expect(csls.size()).toEqual(1)
+      expect(csls.first).toEqual(222)
+
+      expect(csls.remove(222)).toBeTruthy
+      expect(csls.size()).toEqual(0)
+      expect(csls.isEmpty).toBeTruthy
+      expect(csls.remove(333)).toBeFalsy
+      expect{
+          try {
```

Comment:
One would think so. Usually there must be a space before `{`. But there is an exception if it preceded by `(`. Now teach that to Scalastyle ^^